Jenkinsfile

```diff
@@ -27,6 +27,10 @@ try {
             sh "CCACHE_DIR=${CCACHE_DIR} VERSION=${VERSION} build/run make -j${parallel} release"
         }
 
+        stage('Style Check') {
```

Comment:
I agree. I have added while trying to find a way to keep this seperate from the other checks for development. I will remove it since i know the check works